src/pytorch_lightning/lite/lite.py

```diff
@@ -457,7 +458,7 @@ def _check_strategy_support(self, strategy: Optional[Union[str, Strategy]]) -> N
     def _supported_device_types() -> Sequence[_AcceleratorType]:
         return (
             _AcceleratorType.CPU,
-            _AcceleratorType.GPU,
```

Comment:
Actually no, because there is no GPUAccelerator (only the deprecated one replaced by the CUDAAccelerator) anymore, so we don't have a AcceleratorType anymore for this